src/components/NavBar/NavigationBar.tsx

```diff
@@ -1,18 +1,27 @@
 import React, {useState, useEffect} from 'react';
 import './NavigationBar.css';
 import cehatLogo from './CEHATLogo.png';
-import menuIcon from './menuIcon.png';
 import {useTranslation} from 'react-i18next';
+import {FaBars, FaGlobeAmericas} from 'react-icons/fa';
 
 function NavigationBar(): JSX.Element {
```

Comment:
(I know this wasn't your change but can you also add a docstring here please thank you)